### PR TITLE
ft[obstacles]: Adds generic size property

### PR DESCRIPTION
**Pull request by c-salmi**
Useful to generically query the size of a list of obstacle that can have different types.

### CODE REVIEW

**Comment by maxspahn** [comment]
Can you bump the version number in the `pyproject.toml` file, so that it can be directly published to pypi?

File: mpscenes/obstacles/collision_obstacle.py [hunk 1]
```diff
@@ -79,3 +79,6 @@ def movable(self):
 
     def update_bullet_position(self, pybullet, **kwargs):
         pass
+
```
Comment (maxspahn) [inline mpscenes/obstacles/collision_obstacle.py:82]: Can you add the abstractmethod decorator? I think this is needed to enforce that this method is implemented.

File: mpscenes/obstacles/box_obstacle.py [hunk 2]
```diff
@@ -56,6 +56,13 @@ def __init__(self, **kwargs):
         super().__init__(**kwargs)
         self.check_completeness()
 
+    def size(self):
+        return [
+            self.length(),
+            self.width(),
+            self.height(),
+        ]
+
```
Comment (maxspahn) [inline mpscenes/obstacles/box_obstacle.py:65]: Should we already convert that to a numpy array here? Or is this taken care of in the FullSensor later on?

**Review by maxspahn — CHANGES_REQUESTED** [review]
And bump the version.

File: mpscenes/obstacles/box_obstacle.py (hunk 2 above)
Comment (c-salmi) [reply mpscenes/obstacles/box_obstacle.py:65]: It is taken care of in the fullsensor [here](https://github.com/maxspahn/gym_envs_urdf/blob/1c4506df97497f757c6fad219c1685c56162f872/urdfenvs/sensors/full_sensor.py#L74)

File: mpscenes/obstacles/collision_obstacle.py (hunk 1 above)
Comment (c-salmi) [reply mpscenes/obstacles/collision_obstacle.py:82]: :+1:

**Comment by c-salmi** [comment]
The urdf obstacle returns an empty list as size, not sure what to return there

**Review by maxspahn — APPROVED** [review]
The urdf obstacle cannot return much there...

**Pull request merged**